packages/blocks/src/components/question-wrapper/index.js

```diff
@@ -56,6 +56,12 @@ const Content = styled.div`
 			margin-bottom: 0;
 		}
 	}
+
+	&.is-horizontal {
```

Comment:
Technically this works... I wonder though if it's maybe better to create a separate `<Row />` component for this or something? Would make adjusting some of these like `justify-content` easier in the future.  
And this will definitely not cut it for something like a matrix setup.